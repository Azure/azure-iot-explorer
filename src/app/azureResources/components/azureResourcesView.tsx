/***********************************************************
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License
 **********************************************************/
import * as React from 'react';
import { ConnectionStringsViewContainer } from '../../connectionStrings/components/connectionStringsView';

export const AzureResourcesView: React.FC = () => {
       return <ConnectionStringsViewContainer />;
};
