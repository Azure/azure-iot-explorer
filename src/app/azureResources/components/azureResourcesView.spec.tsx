/***********************************************************
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License
 **********************************************************/
import * as React from 'react';
import { shallow } from 'enzyme';
import { AzureResourcesView } from './azureResourcesView';

describe('azureResourcesView', () => {
    it('matches snapshot', () => {

        const routerprops = {
            history: jest.fn() as any, // tslint:disable-line:no-any
            location: jest.fn() as any, // tslint:disable-line:no-any
            match: {
                params: {
                    hostName: 'hostName'
                },
                url: 'currentUrl',
            } as any // tslint:disable-line:no-any
        };
        const wrapper = shallow(<AzureResourcesView {...routerprops} />);
        expect(wrapper).toMatchSnapshot();
    });
});
