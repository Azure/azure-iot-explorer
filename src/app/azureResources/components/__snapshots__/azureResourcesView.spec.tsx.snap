// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`azureResourcesView matches snapshot 1`] = `<Component />`;
